recipes/example/meta.yaml

```diff
@@ -1,6 +1,7 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
 
 # Jinja variables help maintain the recipe as you'll update the version only here.
+{% set org = "simplejson" %}
```

Comment:
I know that I have added org in some of my recipes, but in retrospect, it isn't something that changes often and it usually only in one or two parts of the code base.  I think it is better just remove it from the example.